lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "packages": ["packages/*"],
+  "packages": ["packages/*", "samples/react-app"],
```

Comment:
Looks like this allows the react-app sample to get linked by Lerna. We should do this for the angular-app, vue-app, and sp-webpart samples too :) I made a new issue #595

Reply:
Yeah. I only did the react one to not complicate this PR. Thanks for creating the issue